content/applications/sales/sales/products_prices/returns.rst

```diff
@@ -0,0 +1,100 @@
+===========================
+Process returns and refunds
+===========================
+
+The :guilabel:`Sales` app provides two different ways to process returns based on whether an
+invoice has been sent or not.
+
+Before invoicing
+================
+
+Returns are completed using *Reverse Transfers* when a customer decides to return a product before
+an invoice has been sent or validated.
+
+.. note::
+   In order to use Reverse Transfers, the :guilabel:`Inventory` app must also be installed.
+
+To start a return, open the :guilabel:`Sales` app, navigate to the customer's sales order, and
+click on the :guilabel:`Delivery` smart button to open the associated delivery order.
+
+On the validated delivery order, click :guilabel:`Return` to open the :guilabel:`Reverse Transfer`
+pop-up window. By default, the :guilabel:`Quantity` matches the validated quantities from the
+delivery order. Update the quantities if necessary. Click on the trash icon next to a line item to
+remove it from the return.
+
+.. image:: returns/reverse-transfer-popup.png
+   :align: center
+   :alt: Reverse Transfer Pop-Up.
```

Comment:
- Add `The` before `Reverse Transfer`
- Add quotation marks around `Reverse Transfer`
- Uncapitalize `Pop-Up`
- Add `window` after `pop-up`
- Add a period after `window`